src/internal/dockertestenv/postgres.go

```diff
@@ -48,39 +48,39 @@ func NewTestDBConfig(t testing.TB) serviceenv.ConfigOption {
 		c.PostgresDBName = dbName
 
 		// direct
-		c.PostgresHost = postgresHost()
-		c.PostgresPort = postgresPort
+		c.PostgresHost = PostgresHost()
+		c.PostgresPort = PostgresPort
 		// pg_bouncer
-		c.PGBouncerHost = pgBouncerHost()
-		c.PGBouncerPort = pgBouncerPort
+		c.PGBouncerHost = PGBouncerHost()
+		c.PGBouncerPort = PGBouncerPort
 
 		c.PostgresUser = testutil.DefaultPostgresUser
 	}
 }
 
 func NewTestDB(t testing.TB) *pachsql.DB {
 	opts := NewTestDBOptions(t)
-	return testutil.NewTestDB(t, opts)
+	return testutil.OpenDB(t, opts...)
```

Comment:
This was a bug because both `NewTestDBOptions` and `NewTestDB` call `CreateEphemeralDB`, thereby unintentionally creating two databases instead of one.